package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nexmo-cli",
-  "version": "0.0.12",
+  "version": "0.0.13",
```

Comment:
I prefer not to bump in the PR. Rather let's get the changes merged, and then bump in master explicitly (and tag the release)